### PR TITLE
Lab 6

### PR DESCRIPTION
**Pull request by danylykkk**

### CODE REVIEW

File: PYlab6.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import dependency_injector.containers as containers
```
Comment (polyakovartem) [inline PYlab6.py]: there is no such module in python3 as dependency_injector

File: PYlab6.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+class DataBases(containers.DeclarativeContainer):
+    mysql = providers.Factory(MySQLDB)
+    mongo = providers.Factory(MongoDB)
+
+class CBooks(containers.DeclarativeContainer):
+    mysql = providers.Factory(Books, db=DataBases.mysql)
+    mongo = providers.Factory(Books, db=DataBases.mongo)
+
+if __name__ == '__main__':
+    books_mysql = CBooks.mysql()
```
Comment (polyakovartem) [inline PYlab6.py]: this looks awkward, how is it a dependency injection if you refer to a concrete object instead of abstraction

**Review by polyakovartem — CHANGES_REQUESTED** [review]

File: PYlab6.py (hunk 2 above)
Comment (danylykkk) [reply PYlab6.py]: It is just easier way like in that web page you have sent.

**Comment by danylykkk** [comment]
Review it please)

**Review by polyakovartem — CHANGES_REQUESTED** [review]
now please make it the right way instead of "easy" way

**Review by polyakovartem — APPROVED** [review]

**Pull request closed without merging**